webpack/assets/javascripts/bundle.js

```diff
@@ -4,7 +4,8 @@ require('expose?_!lodash');
 require('expose?jstz!jstz');
 require('jquery.cookie');
 require('./bundle_flot');
+require('script!../../../node_modules/patternfly/dist/js/patternfly.js');
```

Comment:
not sure i follow? it does work for me, however, see previous comment about patternfly gem vs node package.